src/notebook/legifrance.md

```diff
@@ -0,0 +1,158 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.17.3
+kernelspec:
+  name: csplab-concours
+  display_name: CSPLab concours
+  language: python
+---
+
+```{code-cell} ipython3
```

Comment:
By using Myst markdown instead of plain markdown, we lose code-blocks syntax highlighting. That's not ideal for reviews 😬